libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1450,6 +1450,16 @@ void ExpressionCompiler::appendArithmeticOperatorCode(Token::Value _operator, Ty
 		m_context << Instruction::DUP2 << Instruction::ISZERO;
 		m_context.appendConditionalInvalid();
 
+		// Test for (minInt) / (-1)
+		if (c_isSigned && _operator == Token::Div)
+		{
+			m_context.appendInlineAssembly(R"({
+				jumpi(invalidJumpLabel, and(eq(y, exp(2,255)), iszero(add(x, 1))))
```

Comment:
The `invalidJumpLabel` is gone. This should use something like:
```
jumpi(safe, iszero(...))
invalid
safe:
```

Thought not sure if the labels must be unique?